tests/integration_tests/test_configurations.py

```diff
@@ -26,20 +27,21 @@
 CONFIGURATION_NAME = "localhost"
 configuration_id = -1
 
-def test_configurations_clean():
+def test_configurations_clean(caplog):
+    caplog.set_level(logging.INFO)
```

Comment:
I think we can define our own fixture that takes in `caplog` and sets the loglevel once. Then, we can just take in that fixture in each of our tests.